src/Microsoft.ML.Transforms/MissingValueReplacing.cs

```diff
@@ -32,12 +32,10 @@
 
 namespace Microsoft.ML.Transforms
 {
-    // This transform can transform either scalars or vectors (both fixed and variable size),
-    // creating output columns that are identical to the input columns except for replacing NA values
-    // with either the default value, user input, or imputed values (min/max/mean are currently supported).
-    // Imputation modes are supported for vectors both by slot and across all slots.
+    /// <summary>
+    /// <see cref="ITransformer"/> resulting from fitting a <see cref="MissingValueReplacingEstimator"/>.
+    /// </summary>
     // REVIEW: May make sense to implement the transform template interface.
```

Comment:
Should we remove this comment?

Reply:
it's just a comment. It doesn't do any harm, and not gonna showed up in documentation.